helix-front/package.json

```diff
@@ -80,6 +80,7 @@
     "@swimlane/ngx-graph": "8.0.0",
     "ajv": "7.1.0",
     "angulartics2": "^10.1.0",
+    "applicationinsights": "2.3.6",
     "body-parser": "^1.17.2",
```

Comment:
I recently ELR'd the latest version of `applicationinsights` to LinkedIn
Most recent version you can now use is: `2.5.0`

Looking at the Release notes for `applicationinsights` looks like its a lot of bug fixes & quality of life improvements
https://github.com/microsoft/ApplicationInsights-node.js/releases